doc/source/user_guide/indexing.rst

```diff
@@ -1126,7 +1126,82 @@ Mask
 
 .. _indexing.query:
 
-The :meth:`~pandas.DataFrame.query` Method
+The ``Series`` :meth:`~pandas.Series.query` Method
+------------------------------------------
+
+:class:`~pandas.Series` objects also have a :meth:`~pandas.Series.query`
+method that allows for selection using an expression. Perviously, the
+behaviour required the conversion of a ``Series`` object into a 
+``DataFrame`` in order to use the query method. The differences are 
+shown below.
+
+.. ipython:: python
+
+   data = [10, 11, 12, 13, 14, 15]
+   index = [0, 1, 2, 3, 4, 5]
+   series = pd.Series(data, index=index)
+   series
+
+   # previously, converting to DataFrame object
+   series.to_frame().query('index > 2')
+
+   # using query method for series
+   series.query('index > 2')
+
+You can do the same with a named ``index``.
+
+.. ipython:: python
+   
+   data = [10, 11, 12, 13, 14, 15]
+   index = [0, 1, 2, 3, 4, 5]
+   series = pd.Series(data, index=index)
+   series.index.name = 'foo'
+
+   series.query('foo > 2')
+
+The ``query`` method also has a ``inplace`` keyword that lets you modify
+the originail ``Series``. It is set to ``False`` by default. 
```

Comment:
Should be "original"

Reply:
fixed!

Reply:
let me know if the eval stuff looks good. you can look at the output files on the google doc too, i've posted screen shots

Reply:
I suggested a minor change but it looks solid otherwise, thanks for adding it!

Reply:
fixed your suggestion. Thanks!!